### PR TITLE
Fix #330 traversal problem in the portlet_events template when an obj…

### LINKED ISSUE #330: Traversal issues if a content is called `image`
Given a content with `image` id, and a listing in a folder, or in the portlet_event, the traversal goes crazy and completely breaks the server itself (and kills it) with a recursive memory leak:

```
   - Expression: <PathExpr path:'view/heading_link_target'>
   - Names:
      {'args': (),
       'container': <Image at /Plone/de/beispiele/image>,
       'context': <Image at /Plone/de/beispiele/image>,
       'default': <DEFAULT>,
       'here': <Image at /Plone/de/beispiele/image>,
       'loop': {},
       'nothing': None,
       'options': {},
       'repeat': <Products.PageTemplates.Expressions.SafeMapping object at 0x11c3b31c0>,
       'request': <WSGIRequest, URL=http://localhost:8080/Plone/de/folder_listing>,
       'root': <Application at >,
       'template': <Products.Five.browser.pagetemplatefile.ViewPageTemplateFile object at 0x11d689fd0>,
       'traverse_subpath': [],
       'user': <PropertiedUser 'admin'>,
       'view': <plone.app.portlets.portlets.navigation.Renderer object at 0x11d630b20>,
       'views': <Products.Five.browser.pagetemplatefile.ViewMapper object at 0x11d6f6d90>}
  Module zope.tales.expressions, line 250, in __call__
  Module Products.PageTemplates.Expressions, line 196, in _eval
  Module Products.PageTemplates.Expressions, line 126, in render
  Module plone.app.portlets.portlets.navigation, line 242, in heading_link_target
  Module zope.component._api, line 102, in getMultiAdapter
  Module zope.component._api, line 116, in queryMultiAdapter
  Module zope.interface.registry, line 364, in queryMultiAdapter
  Module zope.interface.adapter, line 560, in queryMultiAdapter
  Module zope.interface.adapter, line 560, in <listcomp>
  Module plone.dexterity.content, line 163, in __get__
  Module plone.dexterity.schema, line 99, in decorator
  Module plone.dexterity.schema, line 62, in lookup_fti
  Module zope.globalrequest, line 13, in getRequest
  Module zope.globalrequest.local, line 11, in getLocal
RecursionError: maximum recursion depth exceeded in comparison
2021-05-27 12:11:08,629 ERROR   [CMFCore.MembershipTool:163][waitress-0] Error during wrapUser
Traceback (most recent call last):
  File "/Users/sneridagh/Development/eggs/Products.CMFCore-2.4.8-py3.8.egg/Products/CMFCore/MembershipTool.py", line 159, in wrapUser
    u = mdtool.wrapUser(u)
  File "/Users/sneridagh/Development/eggs/Products.CMFCore-2.4.8-py3.8.egg/Products/CMFCore/MemberDataTool.py", line 196, in wrapUser
    return getMultiAdapter((u, self), IMember)
  File "/Users/sneridagh/Development/eggs/zope.component-4.6.2-py3.8.egg/zope/component/_api.py", line 102, in getMultiAdapter
    adapter = queryMultiAdapter(objects, interface, name, context=context)
  File "/Users/sneridagh/Development/eggs/zope.component-4.6.2-py3.8.egg/zope/component/_api.py", line 116, in queryMultiAdapter
    return sitemanager.queryMultiAdapter(objects, interface, name, default)
  File "/Users/sneridagh/Development/eggs/zope.interface-5.2.0-py3.8-macosx-11-x86_64.egg/zope/interface/registry.py", line 364, in queryMultiAdapter
    return self.adapters.queryMultiAdapter(
  File "/Users/sneridagh/Development/eggs/zope.interface-5.2.0-py3.8-macosx-11-x86_64.egg/zope/interface/adapter.py", line 564, in queryMultiAdapter
    result = factory(*[o.__self__ if isinstance(o, super) else o for o in objects])
  File "/Users/sneridagh/Development/eggs/Products.PlonePAS-6.0.6-py3.8.egg/Products/PlonePAS/tools/memberdata.py", line 234, in __init__
    BaseMemberAdapter.__init__(self, user, tool)
  File "/Users/sneridagh/Development/eggs/Products.CMFCore-2.4.8-py3.8.egg/Products/CMFCore/MemberDataTool.py", line 241, in __init__
    self._md = tool._members.get(id, MemberData(id))
RecursionError: maximum recursion depth exceeded
Fatal Python error: Cannot recover from stack overflow.
Python runtime state: initialized
```

### PR DESCRIPTION
**Pull request by sneridagh**
…ect in a folder is called image

### CODE REVIEW

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

**Comment by sneridagh** [comment]
Maybe we need this also in pre-Plone6 releases.

File: plone/app/event/portlets/portlet_events.pt [hunk 1]
```diff
@@ -25,7 +25,7 @@
                         item_descr    item/description;
                         item_title    item/title;
                         item_location item/location;
-                        item_hasimage item/context/image|nothing"
+                        item_hasimage python:getattr(item.context.aq_base, 'image', None)"
```
Comment (thet) [inline plone/app/event/portlets/portlet_events.pt:28]: :+1:

**Review by thet — APPROVED** [review]
LGTM.

**Pull request merged**